spec/tictactoe/ruby_spec.rb

```diff
@@ -1,6 +1,24 @@
-RSpec.describe Tictactoe do
+require "./lib/tictactoe/game"
```

Comment:
I would give more thought on the name of the test file, `ruby_spec.rb` does not describe what we are testing here.